theme/islandora-manuscript-manuscript.tpl.php

```diff
@@ -5,7 +5,7 @@
  */
 ?>
 <?php if (isset($viewer)): ?>
-  <div id="manuscript-viewer">
+  <div id="manuscript-viewer" class="ia-bookreader">
```

Comment:
This class is specific to the one viewer and should be conditional on viewer configuration.